drupal-org.make

```diff
@@ -66,6 +66,11 @@ projects:
     version: '1.5'
   diff:
     version: '3.3'
+  dkan_dataset_paragraphs:
```

Comment:
Hmm.. why is this stuff here? What branch did you rebase against? This isn't in 7.x-1.x or develop atm.